variables.tf

```diff
@@ -74,7 +74,7 @@ variable "ecs_load_balancers" {
 
 variable "container_definition_json" {
   type        = string
-  description = "The JSON of the task container definition"
+  description = "JSON-encoded list of container definitions for this task. For example: `\"[{ \"name\": \"container1\", ... }, { \"name\": \"container2\", ... }]\"`. See https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDefinition.html and https://github.com/cloudposse/terraform-aws-ecs-container-definition"
```

Comment:
This wording is a bit confusing.

This variable is a string and can take a string of a single task definition in or NOT in list form) or can take a string of multiple task definitions in list form.